api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -151,7 +172,9 @@ def open_soma(
             "consistency."
         )
 
-    return _open_soma(description["soma"], context)
+    locator = _resolve_census_locator(description["soma"], selected_mirror)
+
+    return _open_soma(locator, context)
 
 
 def get_source_h5ad_uri(dataset_id: str, *, census_version: str = DEFAULT_CENSUS_VERSION) -> CensusLocator:
```

Comment:
will this ever need a mirror name as well?

We are moving to the decoupling of these two data sources, but I don't see why they would not be _both_ potentially mirrored, and for the same reasons.   CC: @pablo-gar @brianraymor

Reply:
Discussing with @pablo-gar and @brianraymor  -  it seems possible that we may entirely remove this function in an upcoming release (when we resolve issue #89).

If that happens, all of my comments are moot.  

Given this, I suggest the following approach (@pablo-gar - can you comment on this?):
1. Now:  Mark this method (in the docstring) as to be deprecated in a near-future release.  Otherwise, leave it alone
2. When we fix #89, remove it entirely.  

Assumption: the Census `datasets` table has sufficient information for a user to find the H5AD used to build the Census given access to the Discover REST API.

Reply:
I think this makes sense, but we should confirm with Pablo.

Reply:
Ok so having a conversation with Bruce and Brian we should move forward with the following.

1. Do not mirror h5ads.
2. `download_source_h5ad` and `get_source_h5ad_uri` must still work and always fetch from `us-west-2` since it will the only region available for them.
3. Bruce to to address #89 in the near future.
4. After,`download_source_h5ad` and `get_source_h5ad_uri` are **either** kept but their underlying implementation produces and utilizes Discover's URLs  **OR**  they are deprecated. For now I'm inclined to go with the former, but I'd like to discuss this further.